### PR TITLE
Bug fixing dockerize unomi and update unomi to v1.4.0

### PR DESCRIPTION
**Pull request by aarestu**
1. fix error : entrypoint.sh not Unix Format
2. add ENVs elasticsearch
3. expose port 8102 (Apache Karaf SSH Shell)
4. update unomi to v1.4.0

### CODE REVIEW

**Comment by mikeghen** [comment]
@aarestu, this looks good to me. Thank you so much for this contribution!

**Comment by aarestu** [comment]
> @aarestu, this looks good to me. Thank you so much for this contribution!

My pleasure :)

File: docker/Dockerfile [hunk 1]
```diff
@@ -18,20 +18,30 @@
...
 
-RUN wget http://apache.mirrors.pair.com/incubator/unomi/1.3.0-incubating/unomi-1.3.0-incubating-bin.tar.gz
-RUN tar -xzf unomi-1.3.0-incubating-bin.tar.gz
-RUN mv unomi-1.3.0-incubating/* .
-RUN rm unomi-1.3.0-incubating-bin.tar.gz
-RUN rm -r unomi-1.3.0-incubating
+ENV ELASTICSEARCH_HOST localhost
+ENV ELASTICSEARCH_PORT 9300
+
+ENV UNOMI_VERION "1.4.0"
```
Comment (sergehuber) [inline docker/Dockerfile]: I just noticed there's a small typo in VERION. Should be VERSION. Can you correct that ?

Comment (Taybou) [inline docker/Dockerfile]:
```suggestion
ENV UNOMI_VERSION "1.4.0"
```

File: docker/Dockerfile [hunk 2]
```diff
@@ -18,20 +18,30 @@
...
+
+WORKDIR $UNOMI_HOME
+
+RUN wget http://apache.mirrors.pair.com/unomi/${UNOMI_VERION}/unomi-${UNOMI_VERION}-bin.tar.gz
```
Comment (Taybou) [inline docker/Dockerfile]:
```suggestion
RUN wget http://apache.mirrors.pair.com/unomi/${UNOMI_VERSION}/unomi-${UNOMI_VERSION}-bin.tar.gz
```

File: docker/Dockerfile [hunk 3]
```diff
@@ -18,20 +18,30 @@
...
+
+RUN tar -xzf unomi-${UNOMI_VERION}-bin.tar.gz \
```
Comment (Taybou) [inline docker/Dockerfile]:
```suggestion
RUN tar -xzf unomi-${UNOMI_VERSION}-bin.tar.gz \
```

**Comment by aarestu** [comment]
oke, I apologize for the typo..

thanks, @mikeghen @Taybou

**Comment by sergehuber** [comment]
The PR was just merged, thank you so much for your contribution !

**Comment by aarestu** [comment]
@sergehuber 
My pleasure :)

**Pull request merged**